go/vt/topo/topoproto/flag.go

```diff
@@ -56,21 +49,24 @@ func (ttlv *TabletTypeListValue) Get() any {
 	return *ttlv
 }
 
-// TabletTypeFlag implements the flag.Value interface, for parsing a command-line value into a TabletType.
+// TabletTypeFlag implements the flag.Value and pflag.Value interfaces, for parsing a command-line value into a TabletType.
```

Comment:
Do we really need to keep the `flag.Value` implementation?

Reply:
If we decide no, we can delete `Get()` and change the comments on the remaining funcs.

Reply:
https://github.com/vitessio/vitess/pull/10863#discussion_r931565945